src/tree/box.js

```diff
@@ -8,6 +8,17 @@ export default function box(divBox, o, appendNodes, cardStyling, reRender) {
     return `width:${o.nodeSize.width}px;height:${o.nodeSize.height}px; top:${o.y(p)}px;left:${o.x(p)}px;`;
   }
 
+  function getTooltipStyle(p) {
+    if (p.data.id === 'Root') {
+      return `top:${o.y(p) + 55}px;left:${o.x(p) + 140}px;visibility: visible; opacity: 1;`;
+    }
+    return `width:200px;height:50px; top:${o.y(p) - 25}px;left:${o.x(p)}px;visibility: visible;opacity: 1;`;
```

Comment:
this style is not applied, so the tooltip does not move to the right position right now